### PR TITLE
update the deployment file and use configmap

### PR DESCRIPTION
**Pull request by hchenxa**
update the deployment file and add the sample configmap

### CODE REVIEW

**Comment by andyxning** [comment]
@hchenxa you need to sign the cla first!

**Comment by hchenxa** [comment]
@andyxning the cla was already signed.

File: deployment/node-problem-detector.yaml [hunk 1]
```diff
@@ -16,8 +16,8 @@ spec:
         command:
         - /node-problem-detector
         - --logtostderr
-        - --kernel-monitor=/config/kernel-monitor.json
-        image: k8s.gcr.io/node-problem-detector:v0.2
+        - --system-log-monitors=/config/kernel-monitor.json,/config/docker-monitor.json
+        image: gcr.io/google_containers/node-problem-detector:v0.5.0
```
Comment (andyxning) [inline deployment/node-problem-detector.yaml]: The image should be `k8s.gcr.io/node-problem-detector:v0.5.0`

**Comment by andyxning** [comment]
@hchenxa Sorry for the late. Just one nit. Could you address it?

File: deployment/node-problem-detector.yaml (hunk 1 above)
Comment (andyxning) [reply deployment/node-problem-detector.yaml]: Sorry for the late response to this track. I have not refresh the page. :)

**Pull request merged**